snake game.py

```
@@ -0,0 +1,120 @@
+import pygame
+import time
```

Ruff

`time` imported but unused

Remove unused import: `time`

(F401)